cpu/atmega_common/periph/uart.c

```diff
@@ -98,10 +98,10 @@ static void _set_brr(uart_t uart, uint32_t baudrate)
     }
 #endif
 #if defined(UART_DOUBLE_SPEED)
```

Comment:
We should have a comment here, explaining:
1. This has the effect of rounding the result to the nearest number.
2. It is the same as `util/setbaud.h`
3. and don't change it to what the datasheet says.

Otherwise this will happen again, since at first glance it is not clear at all what this equation does!

Reply:
Thanks, I reverted and commented as per your suggestion.  Thanks for explaining so well :+1: